alertmanager_matrix.go

```diff
@@ -108,15 +123,16 @@ func main() {
 	if err != nil {
 		log.Fatalf("Error connecting to Matrix: %s", err)
 	}
-
+	log.Printf("Connected to Matrix homeserver at %s as %s, and to Alertmanager at %s -- Client instance %+v", homeserver, userID, alertmanagerURL, client)
```

Comment:
This is not exactly true, as the actual connection happens on the first request. Please remove this line.

Reply:
OK.